pythonforandroid/recipes/numpy/__init__.py

```diff
@@ -8,14 +8,11 @@ class NumpyRecipe(CompiledComponentsPythonRecipe):
     version = '1.16.4'
     url = 'https://pypi.python.org/packages/source/n/numpy/numpy-{version}.zip'
     site_packages_name = 'numpy'
+    depends = ['setuptools']
 
     patches = [
         join('patches', 'add_libm_explicitly_to_build.patch'),
-        join('patches', 'do_not_use_system_libs.patch'),
```

Comment:
Can we really get rid of this patch? I wouldn't expect it to be related to the NDK version. If you tested it, are you sure it didn't just work because you don't have the numpy dependency libraries locally?

Reply:
@inclement, you are right, I made a mistake in here :sweat_smile:
I just restored the patch, **¡¡¡thanks for the correction !!!**
:+1: